index.js

```diff
@@ -90,12 +90,13 @@ function getTerms(defaultTerms, terms, exclude) {
 		? loadJson(path.resolve(__dirname, 'terms.json'))
 		: [];
 	const extras = typeof terms === 'string' ? loadJson(terms) : terms;
-	const listTerms = defaults.concat(extras);
 
+	const listTerms = [...(Array.isArray(extras) ? extras : []), ...defaults];
```

Comment:
Extras go first because we use the first matching rule, so that way they have higher priority? Could you add a comment, since it's not super obvious which rules will have higher priority from this place.

Reply:
Yes, I'll add the comment 😉